src/Apps/Artist/Routes/Overview/Components/ArtistArtworkFilter.tsx

```diff
@@ -0,0 +1,206 @@
+import { ArtistArtworkFilter_artist } from "__generated__/ArtistArtworkFilter_artist.graphql"
+import { useTracking } from "Artsy"
+import * as Schema from "Artsy/Analytics/Schema"
+import { BaseArtworkFilter } from "Components/v2/ArtworkFilter"
+import { ArtworkFilterContextProvider } from "Components/v2/ArtworkFilter/ArtworkFilterContext"
+import { updateUrl } from "Components/v2/ArtworkFilter/Utils/urlBuilder"
+import { Location } from "found"
+import React from "react"
+import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
+import { ZeroState } from "./ZeroState"
+
+interface ArtistArtworkFilterProps {
+  artist: ArtistArtworkFilter_artist
+  relay: RelayRefetchProp
+  location: Location
+}
+
+const ArtistArtworkFilter: React.FC<ArtistArtworkFilterProps> = props => {
+  const { location, relay, artist } = props
+  const tracking = useTracking()
+
+  return (
+    <ArtworkFilterContextProvider
+      filters={location.query}
+      sortOptions={[
+        { value: "-decayed_merch", text: "Default" },
+        { value: "-partner_updated_at", text: "Recently updated" },
+        { value: "-published_at", text: "Recently added" },
+        { value: "-year", text: "Artwork year (desc.)" },
+        { value: "year", text: "Artwork year (asc.)" },
+      ]}
+      aggregations={artist.sidebarAggregations.aggregations as any}
+      counts={artist.counts}
+      onChange={updateUrl}
+      onFilterClick={(key, value, filterState) => {
+        tracking.trackEvent({
+          action_type: Schema.ActionType.CommercialFilterParamsChanged,
+          changed: { [key]: value },
+          current: filterState,
+        })
+      }}
+    >
+      <BaseArtworkFilter
+        relay={relay}
```

Comment:
Why is there a need to pass the relay prop around?

Reply:
The artwork filter is an abstraction that fits into various app's existing queries. This is only needed if the schema deviates significantly from the default. (See `/collection/id` communicating with Kaws, for example. )